config/enabled.yml

```diff
@@ -880,7 +880,7 @@ Lint/BlockAlignment:
   Enabled: true
 
 Lint/CircularArgumentReference:
-  Description: "Don't refer to the keyword argument in the default value."
+  Description: "Default values in optional arguments should not refer back to the name of the argument."
```

Comment:
Keyword and optional arguments are different things, so I'd keep the terminology precise.